tests/container

```diff
@@ -135,5 +135,15 @@ for release in $RELEASES; do
     lxc delete -f n1
 done
 
+IMAGE="ubuntu-minimal-daily:24.04"
+
+echo "==> check that we can mount devpts and procfs in unprivileged container (${release})"
+lxc launch "${IMAGE}" u1
+lxc exec u1 -- mkdir /root/proc
+lxc exec u1 -- mount -t proc proc /root/proc
+lxc exec u1 -- mkdir /root/devpts
+lxc exec u1 -- mount -t devpts devpts /root/devpts
+lxc delete -f u1
```

Comment:
Could we have that tested for 22.04 and 24.04 maybe? If so, could that be moved into the for loop?

Reply:
It's up to you. I thought that as these checks and logic behind it completely distro-independent it's better to have it out of the loop to make test a bit faster.

Reply:
Works for me but then `$release` is used out of scope (I know that's not really it but you get the point).

Reply:
ah, yeah. Fixed! Thanks!